lms/envs/common.py

```diff
@@ -3756,7 +3756,7 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
     {
         "NAME": "common.djangoapps.util.password_policy_validators.MinimumLengthValidator",
         "OPTIONS": {
-            "min_length": 2
+            "min_length": 8
```

Comment:
[log] Change of default of `MinimumLengthValidator` from 2 to 8. Also checked against our config at edx.org, and concur in this not causing a breaking change.